Maths/Euclidean Algorithm/C++/factorial.cpp

```diff
@@ -0,0 +1,14 @@
+#include "iostream"
```

Comment:
Why don't you change `"iostream"` to `<iostream>`?

https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename